code/modules/mob/living/simple_animal/abnormality/aleph/melting_love.dm

```diff
@@ -0,0 +1,238 @@
+
+/mob/living/simple_animal/hostile/abnormality/melting_love
+	name = "Melting Love"
+	desc = "A pink slime creature, resembling a female humanoid."
+	icon = 'ModularTegustation/Teguicons/64x64.dmi'
+	icon_state = "melting_love"
+	icon_living = "melting_love"
+	faction = list("slime")
+	speak_emote = list("gurgle")
+	attack_verb_continuous = "glomps"
+	attack_verb_simple = "glomp"
+	health = 3500
+	maxHealth = 3500
+	pixel_x = -16
+	base_pixel_x = -16
+	pixel_y = -16
+	base_pixel_y = -16
+	speed = 2
+	move_to_delay = 5
+	melee_damage_type = BLACK_DAMAGE
+	attack_sound = 'sound/effects/attackblob.ogg'
+	deathsound = 'sound/effects/blobattack.ogg'
+	damage_coeff = list(BRUTE = 1, RED_DAMAGE = -1, WHITE_DAMAGE = 1, BLACK_DAMAGE = 1.5, PALE_DAMAGE = 0.8)
+	obj_damage = 600
+	threat_level = ALEPH_LEVEL
+	del_on_death = FALSE
+	can_breach = TRUE
+	robust_searching = TRUE
+	stat_attack = DEAD
+	start_qliphoth = 3
+	work_chances = list(
+						ABNORMALITY_WORK_INSTINCT = list(20, 20, 30, 40, 40),
+						ABNORMALITY_WORK_INSIGHT = list(40, 40, 40, 45, 45),
+						ABNORMALITY_WORK_ATTACHMENT = list(20, 30, 40, 50, 55),
+						ABNORMALITY_WORK_REPRESSION = list(0, 0, 0, 0, 0),
+						)
+	work_damage_amount = 14
+	work_damage_type = BLACK_DAMAGE
+	ranged = TRUE
+	minimum_distance = 1
+	ranged_cooldown_time = 30
+	projectilesound = 'sound/effects/attackblob.ogg'
+	var/mob/living/carbon/human/gifted_human = null
+	var/norepeating = FALSE
+	var/sanityheal_cooldown = 15 SECONDS
+	var/sanityheal_cooldown_base = 15 SECONDS
+	ego_list = list(/datum/ego_datum/weapon/adoration, /datum/ego_datum/armor/adoration)
+
+/mob/living/simple_animal/hostile/abnormality/melting_love/Initialize()
+	. = ..()
+	bigslime_alive = TRUE //To be sure she doesn't spawn in enraged state
+
+//If you kill big slime first oh boy good luck
+/mob/living/simple_animal/hostile/abnormality/melting_love/Life()
+	. = ..()
+	sanityheal()
+	if(!norepeating)
+		if(!bigslime_alive)
+			melee_damage_lower = 62
+			melee_damage_upper = 80
+			adjustBruteLoss(-maxHealth)
+			projectiletype = /obj/projectile/melting_blob/enraged
+			norepeating = TRUE
+		else
+			melee_damage_lower = 32
+			melee_damage_upper = 40
+			projectiletype = /obj/projectile/melting_blob
+	if(bigslime_alive && norepeating)
+		norepeating = FALSE
+
+//Attacks
+/mob/living/simple_animal/hostile/abnormality/melting_love/AttackingTarget()
+	. = ..()
+	if(. && isliving(target))
```

Comment:
I am fairly sure you only want it to convert humans, so make it check if the target is human instead.
Make sure it doesn't attack dead mobs that aren't human, though.